safe/definitions/post_processors/__init__.py

```diff
@@ -42,6 +47,7 @@
      minimum_needs_post_processors +
```

Comment:
__[flake8]__
* [F405] 'post_processor_size' may be undefined, or defined from star imports: minimum_needs_post_processors, population_post_processors, post_processor_functions, post_processor_inputs, post_processors

<!-- Comment by [SideCI](https://sideci.com) -->